pomodoroTechnique/pomodoroTechnique/Sources/Controllers/BaseViewController.swift

```diff
@@ -8,10 +8,189 @@
 import UIKit
 
 class BaseViewController: UIViewController {
+    
+    // MARK: - Helpers
 
+    private var progress: CGFloat = 0
+    private var visibleProgress: String {
+        let sec = Int(5 * multiplier - progress)
+        let msec = Int(100 - progress.truncatingRemainder(dividingBy: 1) * 100)
+        return String(format: "%02i:%02i", sec, msec == 100 ? 00: msec)
+    }
+    
+    private var timer = Timer()
+    private var isStarted: Bool = false
+    private var isWorkingTime: Bool = true
+    
+    private var multiplier: CGFloat {
+        isWorkingTime ? 5: 1
+    }
+    private var angle: CGFloat {
+        CGFloat.pi * progress / multiplier
+    }
+    private var radius: CGFloat {
+        (circularProgressRect.frame.height > circularProgressRect.frame.width ? circularProgressRect.frame.width: circularProgressRect.frame.height) / 2
+    }
+    
+    private var accentUIColor: UIColor {
+        isWorkingTime ? UIColor.red: UIColor.green
+    }
+    private var accentCGColor: CGColor {
+        accentUIColor.cgColor
+    }
+        
+    // MARK: - UI
+    
+    override var preferredStatusBarStyle : UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    private var timerLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 0, y: 0,
+                                          width: 220,
+                                          height: 80))
+        label.numberOfLines = 1
+        label.font = .monospacedDigitSystemFont(ofSize: 78, weight: .light)
+        label.textAlignment = .center
+        label.text = "25:00"
+        return label
+    }()
+    
+    private var startStopButton: UIButton = {
+        let button = UIButton(frame: CGRect(x: 0, y: 0,
+                                            width: 160,
+                                            height: 160))
+        button.addTarget(self, action: #selector(tapped), for: .touchUpInside)
+        return button
+    }()
+    
+    private var circularProgressRect: UIView = {
+        UIView(frame: CGRect(x: 0, y: 0,
+                             width: 320,
+                             height: 320))
+    }()
+    
+    private var controlImage: UIImage? {
+        if let image = UIImage(named: isStarted ? "pause": "play")?.withRenderingMode(.alwaysTemplate) {
```

Comment:
Не совсем понятно, зачем тебе кастомные картинки плэй и пауза, если они есть в стандартьном наборе с такими же именами
```suggestion
        if let image = UIImage(named: isStarted ? "pause" : "play")?.withRenderingMode(.alwaysTemplate) {
```